default/content/presets/instruct/Command R.json

```diff
@@ -0,0 +1,24 @@
+{
+    "system_prompt": "Write {{char}}'s next reply in a fictional chat between {{char}} and {{user}}. Write 1 reply only in internet RP style, italicize actions, and avoid quotation marks. Use markdown. Be proactive, creative, and drive the plot and conversation forward. Write at least 1 paragraph, up to 4. Always stay in character and avoid repetition.",
```

Comment:
That's why I copied it from chat completion, to stay with that default, and since it feels like a good fit for the Style Guide that explains how the model should respond.

But if you have a newer, better prompt, let me know and I'll update it! Or we just go with the basic "Write {{char}}'s next reply in this fictional roleplay with {{user}}."?

Reply:
Simplified the system prompt as you wished.

The Safety Preamble is necessary and very useful. In the model docs, they use it to censor the model further, but by putting the jailbreak prompt from the chat completions here, it uncensors it very well.

With the same deterministic settings and identical input, with it I got a great response, and without it the usual talk about consent and comfort. So I'd definitely keep it the way it is for best results.